config/api.yml

```diff
@@ -877,6 +877,20 @@
         :identifier: service_tag
       - :name: unassign
         :identifier: service_tag
+  :settings:
+    :description: Settings
+    :identifier: ops_settings
+    :options:
+    - :collection
+    :methods: *70174834086080
+    :collection_actions:
```

Comment:
I'm not sure I understand the distiction between collection/resource here. Is `Settings` not a singleton resource?